temporal-sdk/src/test/java/io/temporal/workflow/LocalAsyncCompletionWorkflowTest.java

```diff
@@ -19,14 +19,10 @@
 
 package io.temporal.workflow;
 
-import io.temporal.activity.Activity;
-import io.temporal.activity.ActivityExecutionContext;
-import io.temporal.activity.ActivityInterface;
-import io.temporal.activity.ActivityMethod;
-import io.temporal.activity.ActivityOptions;
-import io.temporal.client.ActivityCompletionClient;
+import io.temporal.activity.*;
```

Comment:
Unwrap. And change IDE settings to not add .*.

Reply:
It did it before I changed the settings :facepalm: